code/Items/Base/Carriable.cs

```diff
@@ -176,6 +176,14 @@ public virtual void OnCarryDrop( Player dropper )
 		TimeSinceDropped = 0;
 	}
 
+	public override Sound PlaySound( string soundName, string attachment )
+	{
+		if ( Owner.IsValid() )
+			return Owner.PlaySound( soundName, attachment );
+
+		return base.PlaySound( soundName, attachment );
```

Comment:
This should only be `return Sound.FromWorld( sound, Owner.EyePosition )`.

Reply:
It appears to use EyePosition already when playing sounds on the player. Also, using Sound.FromWorld prevents the sound from following the shooter, which is especially noticeable for the shooter while moving.

Carriables can be dropped on the ground. Without the other return, telling the carriable to play a sound would result in no sound being played.